OneCommunityGlobal/HighestGoodNetworkApp
# Logeshwari fix for active/inactive toggle to teams

# Description
![toggle task](https://github.com/user-attachments/assets/750703a4-91e0-4cb7-81e4-a8f48f51e8f9)

## Related PRS (if any):
This frontend PR is related to the development backend PR.

## Main changes explained:
- Added Toggle option for Active /Inactive /SeeAll status of Memebers. 

## How to test:
1. check into current branch
2. do `npm install`, `npm run start:local` to run this PR locally
3. Clear site data/cache
4. Log as Any user with Permissions to Teams Page.
5. Go to Dashboard -> Other Links -> Teams.
6. Select Members of Any Team.
7. Verify the Toggle option available for Team Member status in 1st header column.
8. Toggle option and verify its functionality for different status like 'Active', 'Inactive' and 'See all'.

## Screenshots or videos of changes:

After Changes:
https://github.com/user-attachments/assets/cf2a07d5-d9f7-421e-a477-a20b2366d3dd
